src/service/auth/types.ts

```diff
@@ -0,0 +1,14 @@
+export interface LogInType {
```

Comment:
this file is in a bad directory.

We can change this to `/types/auth/login.ts`